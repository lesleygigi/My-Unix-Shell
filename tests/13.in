 cd   /home
   pwd  
  cd
    exit    
pwd
