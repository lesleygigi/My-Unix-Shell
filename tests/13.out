 cd   /home
   pwd  
/home
  cd
    exit    
